package.json

```
@@ -8,11 +8,12 @@
   "scripts": {
     "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx && sass-lint -v -q",
-    "sass": "gulp sass copy-files",
+    "build": "yarn webpack",
+    "build:prod": " yarn webpack --config webpack.config.js",
```

does this need a production on it?
```suggestion
    "build:prod": " yarn webpack --production --config webpack.config.js",
```

---

@timja what are your thoughts on having a build:dev script that runs webpack-live-reload and the start script concurrently?